gigascience/gigadb-website
# Improve accessibility of keywords input in adminDataset create form

# Pull request for issue: #1573

This is a pull request for the following functionalities:

* Improve keyboard navigation of keywords input in http://gigadb.gigasciencejournal.com:9170/adminDataset/create for chrome and edge
![image](https://github.com/gigascience/gigadb-website/assets/143437854/b8d50f36-eb31-4a64-b010-78f13c1e01f6)

## How to test?

* Navigate to http://gigadb.gigasciencejournal.com:9170/adminDataset/create on Chrome or Edge
* Navigate with keyboard to keywords input
* Add some keywords, navigate to next input and then back to keywords input and verify that keywords input gets visible focus

## How have functionalities been implemented?

* Keyword input styles were tweaked to match better other inputs styles
* Replaced text input by textarea. That solved the issue although to be honest I don't understand why

## Any issues with implementation?

None

## Any changes to automated tests?

None

## Any changes to documentation?

None

## Any technical debt repayment?

None

## Any improvements to CI/CD pipeline?

None
